### PR TITLE
Create daily TUD maintenance job

### PR DESCRIPTION
**Pull request by holdenhinkle**
## Original issue(s)
Closes [ticket #32240](https://app.zenhub.com/workspaces/sprint-board-testing-tools-team-613a3f47e06ba00013d05eed/issues/department-of-veterans-affairs/va.gov-team/32240)

## Description
Creates a daily maintenance job to:
1. Checkin TUD accounts that weren't properly checked in (the 'proper' way is to log a test user out of staging)
2. Effectively mirror the TUD accounts table (minus each account `id` and `password`) in BigQuery for Dashboarding purposes.

The job runs each day at midnight.

## Testing
The BigQuery `tud_accounts` is dropped, then created successfully:

Schema:

Query results:

### CODE REVIEW

**Comment by LindseySaari** [comment]
Is this still in Draft mode?

**Review by LindseySaari — COMMENTED** [review]
I still see "PUTTING THIS BACK IN DRAFT MODE FOR A LITTLE WHILE." at the top of the PR. Is this ready for a rewiew?

**Comment by holdenhinkle** [comment]
> I still see "PUTTING THIS BACK IN DRAFT MODE FOR A LITTLE WHILE." at the top of the PR. Is this ready for a rewiew?

Hey @LindseySaari, Yes, this is ready. I got sidetracked with another issue and forgot to remove that title.

File: app/workers/test_user_dashboard/daily_maintenance.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    def perform
+      checkin_tud_accounts
+      mirror_tud_accounts_in_bigquery
+    end
+
+    private
+
+    def checkin_tud_accounts
+      TestUserDashboard::TudAccount.where.not(checkout_time: nil).each do |account|
+        # rubocop:disable Rails/SkipsModelValidations
```
Comment (LindseySaari) [inline app/workers/test_user_dashboard/daily_maintenance.rb:18]: I think you may want to see the Rubocop docs [here](https://www.rubydoc.info/gems/rubocop/0.51.0/RuboCop/Cop/Rails/SkipsModelValidations) for `Rails/SkipsModelValidations`. I believe you might be able to follow the #good example

**Review by LindseySaari — COMMENTED** [review]
Not sure if you saw my comment here: https://github.com/department-of-veterans-affairs/vets-api/pull/8340#discussion_r741890609

**Review by LindseySaari — APPROVED** [review]

File: app/workers/test_user_dashboard/daily_maintenance.rb (hunk 1 above)
Comment (holdenhinkle) [reply app/workers/test_user_dashboard/daily_maintenance.rb]: Thanks Lindsey. I make an 'update' to that. Har har har :-)

**Review by bramleyjl — APPROVED** [review]

**Pull request merged**